tools/ci_build/build.py

```diff
@@ -290,13 +290,19 @@ def setup_test_data(build_dir, configs):
     # create a shortcut for test models if there is a 'models' folder in build_dir
     if is_windows():
         src_model_dir = os.path.join(build_dir, 'models')
+        if os.path.exists('C:\\local\\models') and not os.path.exists(src_model_dir):
```

Comment:
extract it out as a constant variable

Reply:
May I do it in another PR? Otherwise we'll need to wait another 2 hours. Now all of the pending PRs are waiting this change.

Reply:
How do you update the test data?

Reply:
Much harder than before. Need to rebuild the vm image and re-deploy the VMs.

Reply:
When do you plan on updating models next? Seems like we need to make it as easy as possible to deploy new model sets (without needing to reimage)
For example, we found recently that the tiny_yolov2 models in the model.zip files under opset7 and opset8 are not standard onnx models because they contain ImageScaler op which was removed from onnx. The models in the official onnx model zoo doesn't have this problem so we should replace those models with the official ones.

Reply:
When Ke was here, Ke asked me to do this. Because people often complain our CI build is too slow, so he suggest we should preinstall all the software and avoid re-download the test data 
every time. 

Currently our Windows CI machines occasionally run out of disks because sometimes the machine may get rebooted while it was downloading the test data. So the temp folder won't get cleaned until the same build get schedule the to host again.  Then in most of the cases, the machine will be not usable until someone go there and do the clean up. 

For the tiny_yolov2 issue, we can't replace the model. We need to keep the old one to make sure ImageScaler op still works.  We can add a new one.

Reply:
BTW, I forgot to mention. The data will work in a FULL + incremental way. Disk has the full data, which is about 30+ GBs large.  Azure blob storage has the incremental, which is usually less than 1 GB and it's very easy to update.  onnxruntime can accept multiple input folders.